ScreenControl_Web/src/Configuration/ConfigurationTypes.ts

```diff
@@ -57,6 +58,27 @@ export interface HoverAndHoldInteractionSettings {
     HoverCompleteTimeS: number;
 }
 
+// Class: TouchPlaneInteractionSettings
+// This class is a container for settings that only apply to the TouchPlane interaction. In
+// order to modify these settings of the ScreenControl Service, create an <InteractionConfig>,
+// which contains an instance of this class, modify it as required, and then pass to the service
+// using the <ConfigurationManager>.
+//
+// Like all of the Settings classes found in this file, all members are optional. If you do
+// not modify a member of this class, its value will not change when the instance is sent to
+// ScreenControl Service.
+export interface TouchPlaneInteractionSettings {
+    // Property: TouchPlaneDistanceCM
+    // This determines how far (in cm) the TouchPlane is from the screen surface. This
+    // represents the plane that the user must pass to begin and end a click event.
+    TouchPlaneDistanceCM: number;
```

Comment:
Can we differentiate this and the below further? "TouchPlaneActivationDistance" vs "TouchPlaneStartDistance" or something like that? Having one that is the same as the other but with one more word is a little confusing

Reply:
TouchPlaneStartDistanceCM is only intended to be around until we user test and find a valid distance, this isn't a value we should keep in the UI (or in the config files) as it is hard to explain to users and is unlikely to be wanted to change.

I added it on all sides of the WS just in case we needed it while testing but fully intend to remove it (so isn't overly worth worrying about)

Reply:
That's true but until we do this I'd rather it be less confusing internally 😅